### DISCUSSION TITLE
How do I render HTML from a variable in a safe way?

### DISCUSSION THREAD

**Discussion by slowpoison** [eta-dev/eta#216] [General]
```
var h = '<h1>hi</h1>';
<%= h %>

```
This just prints the HTML as a string. How do I output direct html. Bonus: how do I do it in a safe way?

**Comment by gurgunday** [comment]
You should set h to hi and in your template have: `<h1><%= it.h %></h1>`

You write your html normally and inject HTML-escaped variables using the <%= %> tags. The documentation explains these concepts.
